### DISCUSSION TITLE
How do I dynamically read md file using react.js

### DISCUSSION THREAD

**Discussion by flamecopper** [facebook/docusaurus#10217] [General]
Topic as above, 

const rawData = await import(@site/docs/ios/book1.md');
//         console.log(rawData.default); 

However, it is returning me an mdx object

**Comment by slorber** [comment]
You can get the "source" of any file thanks to Webpack raw-loader

Example documented here:
https://docusaurus.io/docs/markdown-features/react#importing-code-snippets
